src/track/serato/markers.h

```diff
@@ -120,36 +120,30 @@ inline QDebug operator<<(QDebug dbg, const SeratoMarkersEntry& arg) {
                << "isLocked =" << arg.isLocked();
 }
 
-// DTO for storing information from the SeratoMarkers_ tags used by the Serato
-// DJ Pro software.
-//
-// Parsing & Formatting
-// --------------------
-// This class includes functions for formatting and parsing SeratoMarkers_
-// metadata according to the specification:
-// https://github.com/Holzhaus/serato-tags/blob/master/docs/serato_markers_.md
-//
+/// DTO for storing information from the SeratoMarkers_ tags used by the Serato
+/// DJ Pro software.
+///
+/// This class includes functions for formatting and parsing SeratoMarkers_
+/// metadata according to the specification:
+/// https://github.com/Holzhaus/serato-tags/blob/master/docs/serato_markers_.md
 class SeratoMarkers final {
   public:
-    SeratoMarkers() = default;
-    explicit SeratoMarkers(QList<SeratoMarkersEntryPointer> entries)
-            : m_entries(std::move(entries)) {
-    }
+    SeratoMarkers(){};
```

Comment:
`= default` is longer but more specific/explicit then a a custom, empty implementation.